docker/quick_setup/quick_setup.py

```diff
@@ -82,7 +82,30 @@ def get_lostcv(self):
             '          - db-lost\n'
             '          - rabbitmqlost\n\n'
         )
-    
+
+    def get_lostcv_gpu(self):
+        return (
+            '    lost-cv-gpu:\n'
+            '      image: l3pcv/lost-cv-gpu:${LOST_VERSION}\n'
+            '      container_name: lost-cv-gpu\n'
+            '      runtime: nvidia\n'
+            '      command: bash /entrypoint.sh\n'
+            '      env_file:\n'
+            '          - .env\n'
+            '      volumes:\n'
+            '          - ${LOST_DATA}:/home/lost\n'
+            '      restart: always\n'
+            '      environment:\n'
+            '          PYTHONPATH: "/code/backend"\n'
+            '          ENV_NAME: "lost-cv-gpu"\n'
+            '          WORKER_NAME: "lost-cv-gpu-0"\n'
+            '          PY3_INIT: "source /opt/conda/bin/activate lost"\n'
+            '          TF_FORCE_GPU_ALLOW_GROWTH: "true"\n'
```

Comment:
I have added this environmental setting as well. Without this it doesn't run in low GPU memory GPUS (4Gb)